src/App.tsx

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import logo from './logo.svg';
+import Footer from './components/Footer';
 import './App.css';
 import Menu from './components/Menu';
 
 function App() {
-  return <Menu />;
+
+  return (
+    <Footer/>
```

Comment:
Here I'd return both the Menu and Footer, which can be done like:
` <><Menu/><Footer/></>`